### DISCUSSION TITLE
How to change css vars starting with _ (which I assume are private css vars)

### DISCUSSION THREAD

**Discussion by ognjenkljajic-huspy** [mantinedev/mantine#5639] [Q&A] [answered]
in Switch component for example, you can pass `--switch-color`  property when extending the vars of the component in css variables resolver, which will change the background of the switch trail while the switch is "on". 

My intention is also to change the "off" trail background color, in all situations `[data-mantine-color-scheme="light"] .m-{xxxx}` seems to take precedence over any styles that I apply to the component.

![Screenshot 2024-01-25 at 8 17 11 PM](https://github.com/mantinedev/mantine/assets/137264645/14e9c675-0aea-475e-a1ec-a8fdb5d44900)

 So I tried changing `--_switch-bg` in every way possible, and didn't find a way to do so. What's the right way to approach this?
[Sandbox link](https://codesandbox.io/p/sandbox/polished-star-grzcpg)

**Comment by rtivital** [comment, marked as answer]
https://help.mantine.dev/q/private-css-variables

**Reply by ognjenkljajic-huspy** [reply]
Thanks for the swift reply! Damn, I missed the help section.
As already mentioned, `data-mantine-color-scheme` selector was taking precedence over anything else (I thought I'd be able to alter styles by changing private css vars in that case), I managed to solve it by importing `Switch.layer.css` instead of `Switch.css`.
